### DISCUSSION TITLE
Can't open SQL Cipher database after upgrade SQLiteStudio from 3.2.1 to 3.3.3

### DISCUSSION THREAD

**Discussion by wlodek** [pawelsalawa/sqlitestudio#4131] [Q&A]
I can't open database (sql sipher) created in SQLiteStudio 3.2.1 using SQLiteStudio 3.3.3. I have now 2 studios 3.2.1 and 3.3.3. Version 3.3.3 can't open (test function also fails) the database. The password is OK - I checked it 100 times.
Any ideas what to do ?

**Comment by pawelsalawa** [comment]
SQLCipher was updated in SQLiteStudio 3.3, therefore you probably need to enable backward compatibility with your database. You need to do it in the "Cipher configuration" field:
![image](https://user-images.githubusercontent.com/481846/120397407-93315200-c338-11eb-87d4-5873ea021a8d.png)

I'm not sure which version you need to put there (1, 2 or 3). The most recent version (and default) is 4. You need to try and test.

Here's official documentation: https://www.zetetic.net/sqlcipher/sqlcipher-api/#cipher_compatibility

If you want to migrate your database to newer version: https://www.zetetic.net/sqlcipher/sqlcipher-api/#cipher_migrate

**Reply by wlodek** [reply]
Thank You. "PRAGMA cipher_compatibility = 3;" helped.
